_posts/2022-09-28-run-ansible-playbooks.markdown

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title:  "Running Ansible Playbooks on Edge Devices"
+date:   2022-09-28 12:00:00 +0100
+categories: flotta
+author: Gloria Ciavarrini <gciavarrini@redhat.com>
+tags:
+- flotta
+- devices
+- ansible
+
+summary-1:
+---
+There may be cases n which you would like to be able to execute a scripts or commands in a device or on a group of devices.
```

Comment:
I would add that during life cycle of the device we may need to change configuration without rebooting a device which is needed by rpm ostree.

Reply:
Added a sentence. WDYT?